osbs/cli/main.py

```diff
@@ -441,7 +441,10 @@ def cmd_get_token(args, osbs):  # pylint: disable=W0613
 
 
 def cmd_login(args, osbs):
-    osbs.login(args.TOKEN)
+    if args.token and (args.username or args.password):
+        print("Can't use --token and (--user or --pasword).")
+        return
```

Comment:
not sure, based on 406 which is also catching wrong param combinations
it looks a bit weird when printed via logger.error